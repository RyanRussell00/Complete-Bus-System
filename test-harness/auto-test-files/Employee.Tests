# PLEASE READ:
# DO NOT CHANGE THE NAME OF THIS FILE. Query files must end in .Tests and Expected files must end in .Expected
# All statements must end with a semicolon ';'
# Multi-line statements are allowed but the line must not break in the middle of words.
# ONLY USE # or -- FOR COMMENTS.
# The query will expect the output at the same position in the expected file.
#   For example: The 3rd query in this file will expect the 3rd line in the 'Expected' file to be its output.

# SELECT ssn FROM EMPLOYEE WHERE Fname = 'Jeff' AND Lname = 'Mason';
# SELECT ssn FROM EMPLOYEE WHERE Fname = 'Jason' AND Lname = 'Dry';

SELECT * FROM EMPLOYEE;