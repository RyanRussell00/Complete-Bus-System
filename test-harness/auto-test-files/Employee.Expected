### PLEASE READ ###
# DO NOT CHANGE THE NAME OF THIS FILE. Query files must end in .Tests and Expected files must end in .Expected
# Type output EXACTLY as you'd see it on output. No semicolons, no quotes (unless expected),  etc.
# ONLY USE # or -- FOR COMMENTS.
# The queries will expect the output at the same position in the expected file. (line numbers dont matter)
#   For example: The 3rd query in the queries file will expect the 3rd line in this file to be its output.

### FORMATTING FOR EXPECTED ###
# DO NOT END WITH SEMICOLON
# If expecting multiple columns: Use a comma to seperate values, keep expected result on 1 single line
# Strings: use a single quote around the 'string'
# DateTime: If single digit day/month DO NOT put 0 in front.
# DateTime: USE THIS FORMAT EXACTLY: datetime.date(YYYY, (M)M, (D)D) #Do not put 0 in front of single digit Month or Day
# Parentheses around expected output is optional

111222333, 'Jeff', 'L', 'Mason', 'Make This St', 'Lynnwood', 'WA', '98087', datetime.date(2018, 9, 30), 123456789
112233445, 'Jason', 'F', 'Dry', '18115 Campus Way NE', 'Bothell', 'WA', '98011', datetime.date(2019, 12, 20), 123456789
(123456789, 'Mason', 'G', 'Kim', 'Campus', 'Seattle', 'WA', '98195', datetime.date(2019, 9, 5), 123456789)